src/main/kotlin/uk/gov/justice/digital/assessments/services/AssessmentService.kt

```diff
@@ -33,7 +33,7 @@ class AssessmentService(
   private val questionService: QuestionService,
   private val episodeService: EpisodeService,
   private val courtCaseClient: CourtCaseRestClient,
-  private val assessmentUpdateService: AssessmentUpdateService,
+  private val oasysAssessmentUpdateService: OasysAssessmentUpdateService,
   private val offenderService: OffenderService
```

Comment:
this was the trick to make it work, extract the Oasys update in its own service layer. It was not picking it up when called by the same class.